woocommerce/woocommerce
# Fix remote inbox layout overflows the page width

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45516.

Updated the CSS to make sure the remote inbox layout does not overflow the page width. I set the width of action button to `calc(100% - 80px);` so that it still inlines with the Dismiss button

Before:
<img width="296" alt="Screenshot 2024-05-14 at 16 03 48" src="https://github.com/woocommerce/woocommerce/assets/4344253/4fb78ee7-fbed-4578-9013-c0d54d26683a">

After:

<img width="295" alt="Screenshot 2024-05-14 at 16 04 43" src="https://github.com/woocommerce/woocommerce/assets/4344253/1bf6203a-b918-4572-a833-f1e7361ae854">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the WooCommerce admin home page in a mobile layout.
2. Scroll down to the remote inbox notes.
3. Open browser dev tools and inspect the CTA button text
4. Modify the text to a very long string such as `Check out the updated guidelines on product editing page for more information`.
5. Observe that the layout is responsive and the CTA button text does not overflow the page width.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
